py/htm/optimization/README.md

```diff
@@ -35,7 +36,51 @@ Uses PSO ("swarming") with 100 particles, 7 CPU threads, upto 14GB RAM, to optim
 
 `python -m htm.optimization.ae --grid_search "" -n 7 --memory_limit 14 mnist.py` 
 
-Does full exhausive search for all parameters (the `""`) on `mnist.py` script.
+Does full exhaustive search for all parameters (the `""`) on `mnist.py` script.
+
+`python -m htm.optimization.ae --grid_search '["potentialPct"]' -n 7 --memory_limit 14 mnist.py`
+
+Tries several different values for the "potentialPct" parameter.
+
+### Manual search
+
+It is possible to manually specify parameters to evaluate.  While the AE program
+is **not** running, append the following to the lab-report file:
+
+```
+================================================================================
+Modification: ['potentialPct'] = 0.66
+Modification: ['columnDimensions'][1] = 123
+```
+
+The 80 equal signs are significant!  The AE program uses them as section dividers.
+
+Each "modification" line is a snippet of python code which is applied to the
+default parameters.
```

Comment:
nice hack, thanks for explaining it!

Reply:
> Each "modification" line is a snippet of python code which is applied to the
default parameters

- @ctrl-z-9000-times can the manual override be used in conjunction with a running automated search? Eg. I want to auto search for params args1,2, but manually set arg3. Is this a way to do so?

Reply:
First off:  Do not modify the lab-report file while the AE program is running.

You can use any of the search methods, but can only run one of them at a time.  So you can run an automated search over args1,2,   And then at a later time run a manual search over arg3, and they will all show up in the same lab-report.  This way you can do a side-by-side comparison of all of the results.

If you really want to evaluate args1,2,3 all at once, then you can use the command `--all_experiments` which will try every set of parameters in the lab-report file.